hstest/testing/execution/searcher/base_searcher.py

```diff
@@ -175,7 +175,7 @@ def _base_search(self, where_to_search: str) -> RunnableFile:
         return self._simple_search(where_to_search, main_desc='', main_regex='')
 
     def find(self, source: Optional[str]) -> RunnableFile:
-        if source is None:
+        if source in [None, '']:
```

Comment:
Have you checked that this behavior works the same in Java library? (or if there is the same code in Java version)

Reply:
Yes, behavior is same